tests/io/test_structure.py

```diff
@@ -205,7 +205,12 @@ def test_create_structure_without_type():
     ],
 )
 def test_parses_type_case_insensitive(input, expected):
-    structure = Structure(type=input, branchid="branchid", chainage="1")
+    if input.lower() != "compound":
```

Comment:
Introducing such logic in a test seems wrong. Better to write another test or rewrite it with fixtures? What is it that you're testing?

Reply:
See commit below for improvement.